parsl/dataflow/dflow.py

```diff
@@ -510,6 +495,18 @@ def _add_input_deps(self, executor, args, kwargs):
 
         return tuple(newargs), kwargs
 
+    def _add_output_deps(self, executor, args, kwargs, app_fu):
+        logger.debug("Adding output dependencies")
+
+        if not self.check_staging_inhibited(kwargs):
+            outputs = kwargs.get('outputs', [])
+            for f in outputs:
+                if isinstance(f, File) and f.is_remote():
```

Comment:
Is `is_remote` the right description (or does it not matter?) I'm thinking what we really mean here is `is_remote_writeable` (which wouldn't include `http` for example).

Reply:
That's a bit vague. That remote concept goes away in a few PRs time I hope - see big data flow PR - so I haven't been too concerned. As of this PR, I think it means that an http output file would be handed to the stage out code and break there with a reasonable error message.